scripts/ci/install_hadoop.sh

```diff
@@ -23,6 +23,13 @@ echo "export HADOOP_COMMON_HOME=/usr/local/hadoop" >>env.sh
 echo "export HADOOP_HDFS_HOME=/usr/local/hadoop" >>env.sh
 echo "export YARN_HOME=/usr/local/hadoop" >>env.sh
 echo "export HADOOP_COMMON_LIB_NATIVE_DIR=/usr/local/hadoop/lib/native" >>env.sh
+
+# regression on PyArrow==0.16.0: https://issues.apache.org/jira/browse/ARROW-7841
+# it tries to retrieve native library from $HADOOP_HOME directory, instead of
+# `$HADOOP_HOME/lib/native`.
+# force it to search for `libhdfs.so` inside `$HADOOP_HOME/lib/native`
```

Comment:
This regression is fixed in 0.17.0, right? Let's add a note about it as well, so we know that we can drop this line later.

Reply:
fixed, will squash merge (I guess that's what we do now?) Also I presume @skshetry should PR from forks rather than base repo branches?

Reply:
@casperdcl, I wanted to run all of the tests on the upstream branch. PR from forks don't run every test.